### PR TITLE
Implement breadcrumb context menu

### PR DESCRIPTION
**Pull request by JammingBen**
## Description
The last element of the breadcrumb now has a context menu which gives the user the possibility to perform actions on the current folder.

Requires https://github.com/owncloud/owncloud-design-system/pull/1786 to work.

## Related Issue

- Fixes https://github.com/owncloud/web/issues/6030

## Screenshots (if appropriate):

image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [x] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/20431/17/1
The following scenarios passed on retry:
- webUIFilesActionMenu/versions.feature:15

File: packages/web-app-files/src/components/AppBar/AppBar.vue [hunk 1]
```diff
@@ -16,8 +16,13 @@
         id="files-breadcrumb"
         data-testid="files-breadcrumbs"
         class="oc-p-s"
+        :show-contextmenu="true"
```
Comment (fschade) [inline packages/web-app-files/src/components/AppBar/AppBar.vue:19]: is this needed? Can we not just check for the slot.contextMenu in ODS?

File: packages/web-app-files/src/mixins/actions/rename.js [hunk 2]
```diff
@@ -47,12 +46,20 @@ export default {
...
     ...mapActions('Files', ['renameFile']),
 
-    $_rename_trigger({ resources }) {
+    async $_rename_trigger({ resources }) {
+      let parentResources;
+      if (isSameResource(resources[0], this.currentFolder)) {
+        const parentPaths = getParentPaths(resources[0].path, false)
+        const promise = this.$client.files.list(parentPaths[0], 1)
+        parentResources = await promise
+        parentResources = parentResources.map(buildResource)
```
Comment (fschade) [inline packages/web-app-files/src/mixins/actions/rename.js:56]: why not


``` javascript
parentResources = await this.$client.files.list(parentPaths[0], 1)
parentResources = parentResources.map(buildResource)
```

**Review by fschade — APPROVED** [review]
pretty clean, just two optional minor findings, else LGTM

File: packages/web-app-files/src/components/AppBar/AppBar.vue (hunk 1 above)
Comment (JammingBen) [reply packages/web-app-files/src/components/AppBar/AppBar.vue:19]: Good idea, I'll check (and update) the ODS

**Comment by kulmann** [comment]
Could you update the design system in the runtime `package.json` to https://github.com/owncloud/owncloud-design-system/releases/tag/v11.3.0-rc1 ? it's just an RC but we're fine with getting that reference merged and at the end of the sprint update to a real ODS release.

**Review by kulmann — APPROVED** [review]
Awesome! 😍

**Pull request merged**